ops/ccs-ops-misc/scripts/bfd-monthly-inventory

```diff
@@ -0,0 +1,288 @@
+#!/usr/local/bin/bash
+set -eo pipefail
+
+# global variables
+PROGNAME=${0##*/}
+INVENTORY_DIR=~/Desktop/monthly_inventory
+APPS_DIR=${APPS_DIR:-../../../apps}
```

Comment:
first time I ran it I was not in the script directory and so this was populated with a bad path and I got an error. might be nice to support running from any directory

Reply:
To simplify things, I was planning on having folks (likely me) run it from the script directory as it needs to build+get dependencies in /apps. So it _defaults_ to `../../../apps`. Any env var you see in the form of 
```shell
FOO="${FOO:-bar}" 
```
means 'set FOO to the value of the environment variable $FOO if it exists, else make the default bar'

So if you run it from outside the script dir, you would need to update the $APPS_DIR env var. Ie:
```shell
APPS_DIR=~/workspace/bfd/apps ./monthly_inventory
```

I will make INVENTORY_DIR default to ~/Desktop/monthly_inventory as well.

Reply:
Fixed and updated --help to show how to do this